### DISCUSSION TITLE
Issues with Xcode actualization

### DISCUSSION THREAD

**Discussion by edgaresz** [firebase/firebase-ios-sdk#13698] [Q&A]
Recently I upgrade mi Xcode app with Sequoia and after that my flutter app can't run in the simulator and show me this error in the logs:

**Lexical or Preprocessor Issue (Xcode): Include of non-modular header inside
framework module 'firebase_messaging.FLTFirebaseMessagingPlugin'**

I tried updating cocoapods and all the firebase services but I get the same error in the new versions
Can someone pls give me a hand in this? Im codign in flutter

**Comment by paulb777** [comment]
Firebase Podfiles must include `use_frameworks!` or `use_frameworks! :linkage => :static`. `use_modular_headers!` is not supported.

This is being discussed for flutter at https://github.com/firebase/flutterfire/issues/13323

**Reply by farhansadikgalib** [reply]
yeah
